presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/PredicatePushDown.java

```diff
@@ -206,6 +207,29 @@ public PlanNode visitExchange(ExchangeNode node, RewriteContext<Expression> cont
             return node;
         }
 
+        @Override
+        public PlanNode visitWindow(WindowNode node, RewriteContext<Expression> context)
+        {
+            List<Symbol> partitionSymbols = node.getPartitionBy();
+
+            // This could be broader. We can push down conjucts if they are constant for all rows in a window partition. The simplest way to guarantee this is if the conjucts are
```

Comment:
make it `TODO`

Reply:
> We can push down conjucts if they are constant for all rows in a window partition.

You probably mean that symbols used in conjuncts are constant for all rows in window partition.
Please update comment

Reply:
Make comment shorter, break line after `for all rows in a window partition.`